### PR TITLE
Add utm_campaign fields to urls

### PR DESCRIPTION
**Pull request by sherpan**
## Details
also strips the from_llm flag 
## Issues

Resolves #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,40 +27,40 @@ Confidently evaluate, test and monitor LLM applications. 
...
+    * **CI/CD integration**: Run evaluations as part of your CI/CD pipeline using our [PyTest integration](https://www.comet.com/docs/opik/testing/pytest_integration/?utm_source=opik&utm_medium=github&utm_content=pytest_link&utm_campaign=opik)
 
 * **Production Monitoring**: Monitor your LLM application in production and easily close the feedback loop by adding error traces to your evaluation datasets.
 
 <br>
 
 ## 🛠️ Installation
 Opik is available as a fully open source local installation or using Comet.com as a hosted solution.
-The easiest way to get started with Opik is by creating a free Comet account at [comet.com](https://www.comet.com/signup?from=llm?utm_source=opik&utm_medium=github&utm_content=install).
+The easiest way to get started with Opik is by creating a free Comet account at [comet.com](https://www.comet.com/signup?utm_source=opik&utm_medium=github&utm_content=install&utm_campaign=opik).
```
Comment (jverre) [inline README.md]: @sherpan Why did we remove the `from=llm` ? What that on purpose ?

Comment (Nimrod007) [reply README.md]: from=llm is mandatory. please keep in

File: apps/opik-documentation/documentation/docs/cookbook/langchain.ipynb [hunk 2]
```diff
@@ -21,9 +21,9 @@
    "source": [
     "## Creating an account on Comet.com\n",
     "\n",
-    "[Comet](https://www.comet.com/site?from=llm&utm_source=opik&utm_medium=colab&utm_content=langchain) provides a hosted version of the Opik platform, [simply create an account](https://www.comet.com/signup?from=llm&utm_source=opik&utm_medium=colab&utm_content=langchain) and grab you API Key.\n",
+    "[Comet](https://www.comet.com/site?utm_source=opik&utm_medium=colab&utm_content=langchain&utm_campaign=opik) provides a hosted version of the Opik platform, [simply create an account](https://www.comet.com/signup?utm_source=opik&utm_medium=colab&utm_content=langchain&utm_campaign=opik) and grab you API Key.\n",
```
Comment (Nimrod007) [inline apps/opik-documentation/documentation/docs/cookbook/langchain.ipynb]: signup calls need to have from=llm in them

File: README.md [hunk 3]
```diff
@@ -27,40 +27,40 @@ Confidently evaluate, test and monitor LLM applications. 
...
 
 <br>
 
 You can use Opik for:
 * **Development:**
-  * **Tracing:** Track all LLM calls and traces during development and production ([Quickstart](https://www.comet.com/docs/opik/quickstart/?utm_source=opik&utm_medium=github&utm_content=quickstart_link), [Integrations](https://www.comet.com/docs/opik/integrations/overview/?utm_source=opik&utm_medium=github&utm_content=integrations_link)
-  * **Annotations:** Annotate your LLM calls by logging feedback scores using the [Python SDK](https://www.comet.com/docs/opik/tracing/annotate_traces/#annotating-traces-and-spans-using-the-sdk?utm_source=opik&utm_medium=github&utm_content=sdk_link) or the [UI](https://www.comet.com/docs/opik/tracing/annotate_traces/#annotating-traces-through-the-ui?utm_source=opik&utm_medium=github&utm_content=ui_link). 
+  * **Tracing:** Track all LLM calls and traces during development and production ([Quickstart](https://www.comet.com/docs/opik/quickstart/?utm_source=opik&utm_medium=github&utm_content=quickstart_link&utm_campaign=opik), [Integrations](https://www.comet.com/docs/opik/integrations/overview/?utm_source=opik&utm_medium=github&utm_content=integrations_link&utm_campaign=opik)
```
Comment (jverre) [inline README.md]:
```suggestion
  * **Tracing:** Track all LLM calls and traces during development and production ([Quickstart](https://www.comet.com/docs/opik/quickstart/?utm_source=opik&utm_medium=github&utm_content=quickstart_link&utm_campaign=opik), [Integrations](https://www.comet.com/docs/opik/tracing/integrations/overview/?utm_source=opik&utm_medium=github&utm_content=integrations_link&utm_campaign=opik)
```

Fixes broken tracing link

File: apps/opik-documentation/documentation/docs/cookbook/evaluate_hallucination_metric.ipynb [hunk 4]
```diff
@@ -15,9 +15,9 @@
    "source": [
     "## Creating an account on Comet.com\n",
     "\n",
-    "[Comet](https://www.comet.com/site?from=llm&utm_source=opik&utm_medium=colab&utm_content=eval_hall) provides a hosted version of the Opik platform, [simply create an account](https://www.comet.com/signup?from=llm&utm_source=opik&utm_medium=colab&utm_content=eval_hall) and grab you API Key.\n",
+    "[Comet](https://www.comet.com/site/?utm_source=opik&utm_medium=colab&utm_content=eval_hall&utm_campaign=opik) provides a hosted version of the Opik platform, [simply create an account](https://www.comet.com/signup?tm_source=opik&utm_medium=colab&utm_content=eval_hall&utm_campaign=opik) and grab you API Key.\n",
```
Comment (Nimrod007) [inline apps/opik-documentation/documentation/docs/cookbook/evaluate_hallucination_metric.ipynb]: from=llm missing in all signup calls

**Review by Nimrod007 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sherpan) [reply README.md]: @jverre @Nimrod007 we were getting instances where segment wasn't picking up the utm_source=opik when he had from=llm (tagged you in a slack example). do we want to from=llm? to be on every link in the repo?

**Comment by sherpan** [comment]
@Nimrod007 @jverre all links to comet should now have the from=llm parameter

**Review by Nimrod007 — APPROVED** [review]

**Review by Nimrod007 — APPROVED** [review]

**Pull request merged**